recipe/meta.yaml

```diff
@@ -20,7 +20,7 @@ build:
 requirements:
   host:
     - git
-    - python >=3.9
+    - python >=3.9,<3.12
```

Comment:
Bit sad about this, but I guess we can always build a new package once the issues upstream have been fixed.

Reply:
Yeah. :shrug:

Reply:
that's gonna take two packages iterations, gents: once https://github.com/cedadev/MyProxyClient/pull/20 is merged we'll have to get a new version of MyProxyClient, then, with that, `esgf-pyclient` should work fine, but we'll have to unpin and rebuild - maybe for the main release? Although I got doubts Tool will work fine, I already know `aiohttp` is still cranky about 3.12